src/com_tjvendors/site/models/vendor.php

```diff
@@ -161,7 +161,35 @@ public function getItem($pk = null)
 		$vendorXref = Table::getInstance('VendorClientXref', 'TjvendorsTable');
 		$vendorXref->load(array('vendor_id' => $item->vendor_id));
 		$item->params = $vendorXref->params;
+		
+		$TjGeoHelper = JPATH_ROOT . '/components/com_tjfields/helpers/geo.php';
 
+		if (!class_exists('TjGeoHelper'))
+		{
+			JLoader::register('TjGeoHelper', $TjGeoHelper);
+			JLoader::load('TjGeoHelper');
+		}
+
+		$tjGeoHelperObj = new TjGeoHelper;
+		
+		if ($item->country)
+		{
+			$item->country_name = $tjGeoHelperObj->getCountryNameFromId((int) $item->country);
```

Comment:
@deepgawade89 If you have written getCountry method in vendor utilities, why you have loaded TJField file here?